src/pages/en/guides/server-side-rendering.md

```diff
@@ -12,9 +12,18 @@ i18nReady: true
 
 ## Enabling SSR in Your Project
 
-To enable SSR you need to use an adapter. This is because SSR requires a server _runtime_: the environment that runs your server-side code. This runtime provides an API that your server-side code can use.
+To get started, enable SSR features during development with the `output: server` configuration option:
```

Comment:
```suggestion
To get started, enable SSR features in development mode with the `output: server` configuration option:
```
I don't know why, but "during development" isn't working for me.

To get started developing in SSR mode, add the `output: server` configuration option:  ???

Reply:
I like your first suggestion! `development mode` is definitely more clear than what I had